posts/2021-01-06-ocp-log-forwarding.adoc

```diff
@@ -324,9 +324,19 @@ Proceed to the `Review` section and submit. Copy the generated token value to us
 +
 
 
-== Integrating Open Liberty JSON Logs
+== Integrating Open Liberty Logs
 
-Since Open Liberty application pods output logs in JSON format, you must enable Fluentd to parse the JSON fields from the message body.
+Open Liberty has the capability to emit logs to standard out, or in Liberty terms _console output_, in either _JSON_ or _simple_ (plain text) format. These are the logs that the OpenShift platform consumes and redirects to configured destinations. When emitting logs in simple format only message logs are included. The intention is that the output is for human consumption. With JSON formatted output you can configure any type of log to output such as message logs, trace logs, ffdc logs, access logs and audit logs.  The intention is to consolidate all the logging data into a single stream in a mapped format. This logging strategy is optimized for a containerized environment. Ultimately this allows for the log data to be easily indexed and queried by monitoring tools down stream. For example, the JSON data can be leveraged to create meaningful visualizations on log analysis dashboards. The Open Liberty team has already created publicly available dashboards for Kibana and Splunk, but you can modify or create your own dashboards to your monitoring needs. Note that you are still able to use Kibana or Splunk without JSON logging, but without the benefit of being able to leverage all log types and data and you will be limited in terms of querying capabilities.
```

Comment:
Suggested edit:

Open Liberty can emit logs to standard out, or in Liberty terms _console output_, in either _JSON_ or _simple_ (plain text) format. The OpenShift platform consumes and redirects these logs to configured destinations. When emitting logs in simple format, only message logs are included and the output is intended for human consumption. With JSON formatted output, you can configure any type of log to output, such as message logs, trace logs, ffdc logs, access logs, and audit logs. The intention of JSON formatted logs is to consolidate all the logging data into a single stream and in a mapped format. This logging strategy is optimized for a containerized environment. Ultimately, JSON formatted logs allow for the log data to be easily indexed and queried by downstream monitoring tools. For example, the JSON data can be used to create meaningful visualizations on log analysis dashboards. The Open Liberty team provides publicly available dashboards for Kibana and Splunk, but you can modify or create your own dashboards to your monitoring needs. Although you can use Kibana or Splunk without JSON logging, your querying capabilities are limited because you are not able to use all log types and data.